### PR TITLE
View aggregation examples modified to remove unsupported things

### PR DESCRIPTION
**Pull request by cijothomas**
The current spec for View does not allow specifying temporality (cumulative/delta) on aggregation, and it is only allowed along with MetricReader. This PR is modifying the examples to remove the unsupported capabilities, to avoid confusion.

### CODE REVIEW

**Review by bogdandrutu — APPROVED** [review]

**Review by MrAlias — APPROVED** [review]

**Review by jmacd — APPROVED** [review]

File: specification/metrics/sdk.md [hunk 1]
```diff
@@ -253,17 +253,17 @@ meter_provider
 ```python
 # Counter X will be exported as cumulative sum
 meter_provider
-    .add_view("X", aggregation=SumAggregation(CUMULATIVE))
-    .add_metric_reader(PeriodicExportingMetricReader(ConsoleExporter()))
+    .add_view("X", aggregation=SumAggregation())
+    .add_metric_reader(PeriodicExportingMetricReader(AggregationTemporality.CUMULATIVE, ConsoleExporter()))
```
Comment (jmacd) [inline specification/metrics/sdk.md:257]:
```suggestion
    // Export synchronous instruments using Cumulative temporality
    .add_metric_reader(PeriodicExportingMetricReader(AggregationTemporality.CUMULATIVE, ConsoleExporter()))
```

File: specification/metrics/sdk.md [hunk 2]
```diff
@@ -253,17 +253,17 @@ meter_provider
...
 
 ```python
 # Counter X will be exported as delta sum
 # Histogram Y and Gauge Z will be exported with 2 attributes (a and b)
 meter_provider
-    .add_view("X", aggregation=SumAggregation(DELTA))
+    .add_view("X", aggregation=SumAggregation())
     .add_view("*", attribute_keys=["a", "b"])
-    .add_metric_reader(PeriodicExportingMetricReader(ConsoleExporter()))
+    .add_metric_reader(PeriodicExportingMetricReader(AggregationTemporality.DELTA, ConsoleExporter()))
```
Comment (jmacd) [inline specification/metrics/sdk.md:266]:
```suggestion
     // Export synchronous instruments using Delta temporality
    .add_metric_reader(PeriodicExportingMetricReader(AggregationTemporality.DELTA, ConsoleExporter()))
```

**Review by jmacd — DISMISSED** [review]
I'd like it to be clear how the reader should interpret the first argument of the reader. I want it to explain this is the treatment for synchronous instruments, which the SDK has control over. Asynchronous instruments should be exported in their natural temporality, and we do not want to encourage anyone to expect Cumulative-to-Delta conversion inside the SDK.

**Comment by cijothomas** [comment]
> Asynchronous instruments should be exported in their natural temporality, and we do not want to encourage anyone to expect Cumulative-to-Delta conversion inside the S

Don't think this example need to talk about it. There is spec already existing here https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#metricreader which talk about temporality, cumulative->delta conversions etc.

**Comment by jmacd** [comment]
> Don't think this example need to talk about it.

Later in the SDK specification, we write:

> The SDK SHOULD provide a way to allow the preferred [Aggregation
Temporality](./datamodel.md#temporality) to be specified for a `MetricReader`
instance during the setup

But the reader will not have reached that point in the specification yet, and I want it to be clear what this means.

I changed my suggestion from "using" to "preferring" which is more correct.

**Comment by jmacd** [comment]
See #2314

**Pull request merged**